go-chaos/internal/k8helper.go

```diff
@@ -29,12 +30,13 @@ import (
 )
 
 type K8Client struct {
-	ClientConfig clientcmd.ClientConfig
-	Clientset    kubernetes.Interface
+	ClientConfig  clientcmd.ClientConfig
+	DynamicClient dynamic.Interface
```

Comment:
:question: For what is this?

Reply:
we need a dynamic client to work with the SaaS CRDs. The normal k8s client we used for everything else only supports well known resources and can't be used for CRDs that it doesn't know about.

Reply:
Cool thanks :+1: